projects/ion/src/lib/indicator/indicator.component.html

```diff
@@ -0,0 +1,54 @@
+<div data-testid="ion-indicator" class="container">
+  <header class="title">
+    <h1 data-testid="ion-indicator-title">{{ title }}</h1>
+    <ion-icon
+      data-testid="ion-indicator-tooltip"
+      *ngIf="tooltipText"
+      type="exclamation-rounded"
+      size="16"
+      ionTooltip
+      ionTooltipPosition="bottomLeft"
+      ionTooltipColorScheme="dark"
+      ionTooltipTrigger="hover"
+      [ionTooltipTitle]="tooltipText"
+      [ionTooltipArrowPointAtCenter]="true"
+    >
+    </ion-icon>
+  </header>
+  <div class="content">
+    <h4 data-testid="ion-indicator-value">{{ value }}</h4>
+    <p data-testid="ion-indicator-second-value">{{ secondValue }}</p>
+  </div>
```

Comment:
I think that when there is no footer, content should fill in the space. Indicator always will have height at 120px.
![image](https://user-images.githubusercontent.com/57760325/217242766-bc3f3723-a7db-4f18-8b8e-6326d862d7a6.png)